src/sensor.cpp

```diff
@@ -440,6 +441,10 @@ namespace librealsense
         {
             As<librealsense::global_time_interface>(_owner)->enable_time_diff_keeper(true);
         }
+        if (Is<librealsense::device_calibration>(_owner))
+        {
+            As<librealsense::device_calibration>(_owner)->trigger_device_calibration(RS2_CALIBRATION_THERMAL);
```

Comment:
I think this should be in depth sensor start for D400, not here.
I don't think this should go under the `device_calibration` interface, as I see no use-case where the user can "trigger" the calibration -- it's a running process and not a triggerable operation.

Reply:
Note that `l500_device` inherits from `device_calibration` and will throw.

Reply:
Refactored